feat(blog): add blog for December (#60)
mun-lang/website: content/posts/2021-01-05-this-month-december.md

---
title: "This Month in Mun - December 2020"
author: "The Mun Team"

> excerpt: "December was again a slow month, holidays and day jobs have kept the core developers under wraps."

Reviewer: ```suggestion
excerpt: "December - again - was a slow month; holidays and day jobs keeping the core developers busy."
```

---

> December was again a slow month, holidays and day jobs have kept the core developers under wraps. We 

Reviewer: ```suggestion
December - again - was a slow month; holidays and day jobs keeping the core developers busy. We 
```

Reviewer: "under wraps" means "secret"

did finally manage to work together during the holiday break.

> As stated in our [last months update][lmu] we are looking into the possibility of funding a third 

Reviewer: ```suggestion
As stated in [last month's update][lmu] we are looking into the possibility of funding a third 
```

[lmu]: https://mun-lang.org/blog/2020/12/07/this-month-november/

#### Tell me more!

As the reward is commensurate of an internship allowance, we're targetting a similar audience. So,
if you've been working on your own Rust projects and want to take the next step on a bigger,
community-driven project; or have always been interested in honing your compiler, language, or
garbage collection skills; this is an opportunity to do so alongside two industry veterans. An added
bonus is that all of our code is - and will always remain - open-source, so you can show off all of
your work on Mun during future job interviews.

If you are interested in helping develop Mun - but are not sure where to start - feel free to
take a look at our [*good first issues*][gfi] or reach out to us on [Discord][discord],
[Twitter][twi], or the *Contact Us* link at the bottom of our [website](https://mun-lang.org).

[gfi]: https://github.com/mun-lang/mun/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
[twi]: https://twitter.com/munlangorg

#### Mun v0.3 progress

* **feature: use statements language support** [[PR#290](https://github.com/mun-lang/mun/pull/290)]

>   The language side of things is about to be wrapped up to support multi-file projects with the 

Reviewer: ```suggestion
  The language side of things to support multi-file projects is about to be wrapped up with the 
```

>   introduction of `use` statements. We implemented `use` statements very similar to how they work in 

Reviewer: ```suggestion
  introduction of `use` statements. We implemented `use` statements similar to how they work in 
```

  Rust, including support for wildcard imports.

  ```mun
  use package::foo::Bar;
  use package::bar::{self, Baz, *};
  ```

>   The next steps for our project management epic are to support runtime linking between assemblies.

Reviewer: ```suggestion
  The next step for our project management epic is to support runtime linking between assemblies.
```

* **bump: rust 1.49** [[PR#291](https://github.com/mun-lang/mun/pull/291)] and **bump: rust 1.48** [[PR#289](https://github.com/mun-lang/mun/pull/289)]

>   We update our code base to work with the latest versions of Rust. This also always includes all 

Reviewer: ```suggestion
  We updated our codebase to work with the latest versions of Rust. This also included fixes for new
```
